src/scripting/StelScriptMgr.cpp

```diff
@@ -794,10 +794,8 @@ bool StelScriptMgr::runPreprocessedScript(const QString &preprocessedScript, con
 bool StelScriptMgr::runScript(const QString& fileName, const QString& includePath)
 {
 	QString preprocessedScript;
-	if (prepareScript(preprocessedScript,fileName,includePath))
-		return runPreprocessedScript(preprocessedScript,fileName);
-	else
-		return false;
+	prepareScript(preprocessedScript,fileName,includePath);
```

Comment:
prepareScript() returns false if preprocessing fails. In this case, the preprocessedScript should not be run.